IdentityService/DAL/Repository.cs

```diff
@@ -4,11 +4,10 @@
 
     public Repository(SQLiteConnection connection) => _connection = connection;
 
-    public Task<IEnumerable<Permission>> Login(string username, string hashedPassword) => _connection.QueryAsync<Permission>("""
-        SELECT Permissions.Permission
+    public Task<IEnumerable<UserPermissionPairDto>> Login(string username, string hashedPassword) => _connection.QueryAsync<UserPermissionPairDto>("""
+        SELECT Users.Id, Permissions.Permission
         FROM Users
-        JOIN Roles ON Users.RoleId = Roles.Id
-        JOIN Permissions ON Roles.Id = Permissions.RoleId
+        JOIN Permissions ON Users.RoleId = Permissions.RoleId
```

Comment:
Just wondering, if user don't have role/permissions, does it mean that we would return to the user message that this user is not found?

Reply:
Fixed